docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst

```diff
@@ -310,6 +310,26 @@ interacts with PBS, we recommend the following steps:
    the ``SIGTERM``. If after that period of time the job has not terminated, then send a ``SIGKILL``
    to forcibly release all resources.
 
+-  Enable PBS to store job history.
+
+   In order for the Determined launcher to detect successful job completion, PBS must be configured
```

Comment:
Don't we call it the HPC launcher elsewhere on this page?

I don't believe we use phrase "Determined launcher".

Did you run your new text through grammerly.com?

Reply:
Lol

Reply:
Also I think we want this as the first bullet in the PBS requirements -- they are ordered in terms of importance.